### PR TITLE
improve start time and duration display for finished speakers

### PR DESCRIPTION
**Pull request by MaximilianKrambach**
new display for 'finished speakers' using the locale settings, and adding a duration of the speech.

Geman:
```
2. John Doe (No. 2345) (Startzeit: 4.2.2019, 12:38:27, Dauer 2:06m)
```

English:
```
2. John Doe (No. 2345) (Start time: 2/4/2019, 12:38:27 PM, Duration 2:06m)
```

### CODE REVIEW

**Comment by MaximilianKrambach** [comment]
changed @emanuelschuetze

**Comment by MaximilianKrambach** [comment]
sorry, I didn't think of the others force-pushing, and so accidentally destroyed emanuels' sytlistic changes. I tried to reinsert them where I could remember.

File: client/src/app/site/agenda/components/speaker-list/speaker-list.component.ts [hunk 1]
```diff
@@ -111,7 +112,8 @@ export class SpeakerListComponent extends BaseViewComponent implements OnInit {
         private itemRepo: AgendaRepositoryService,
         private op: OperatorService,
         private promptService: PromptService,
-        private currentListOfSpeakersService: CurrentListOfSpeakersSlideService
+        private currentListOfSpeakersService: CurrentListOfSpeakersSlideService,
+        private durationService: DurationService
```
Comment (tsiegleauq) [inline client/src/app/site/agenda/components/speaker-list/speaker-list.component.ts]: missing doc

**Comment by emanuelschuetze** [comment]
I added my last (missing) changes and rebased. Ready for merge...

**Pull request merged**